app/core/credentials.go

```diff
@@ -42,6 +42,12 @@ func (m *MangaDesk) loadCredentials() error {
 // If they are, then read it, and attempt to refresh the token.
 // Will return error if any steps fail (no stored credentials, authentication failed).
 func (m *MangaDesk) restoreSession() error {
+	// We do not need to wait/show a session-related messages
+	// if the user logs in as guest.
+	if !m.Config.GuestLogin {
```

Comment:
sorry if i am wrong about this since i dont have the repo readily available right now, but i think it should be:
```suggestion
	if m.Config.GuestLogin {
```

Reply:
Changed!